workflows/phaseimpute/main.nf

```diff
@@ -103,32 +102,35 @@ workflow PHASEIMPUTE {
     if (params.steps.split(',').contains("simulate") || params.steps.split(',').contains("all")) {
         // Test if the input are all bam files
         getAllFilesExtension(ch_input_sim)
-            .map{ if (it != "bam") {
-                error "All input files must be in BAM format to perform simulation"
+            .map{ if (it != "bam" & it != "cram") {
+                error "All input files must be in the same format, either BAM or CRAM, to perform simulation"
```

Comment:
Are you sure?
Did you try with a mixture of BAM/CRAM?

Reply:
Files need to be all bam or all cram for the moment.
Do we want to allow both at the same time ?

Reply:
No idea, that's more a question for you I guess

Reply:
Maybe just create an issue so you remember to think about it later on